torch/onnx/symbolic_opset9.py

```diff
@@ -732,7 +746,9 @@ def _reduce_op_symbolic(onnx_op_name, allow_multi_dim_support=True):
     @_beartype.beartype
     def symbolic(g, self, dim=None, keepdim=None):
         self = _maybe_cast_reduce_op_input(g, self)
-        if dim is None:
+        if dim is None or dim == tuple():
```

Comment:
```suggestion
        if not dim:
```

Reply:
I think `not dim` makes readers think that it is a mistakenly written None check. Writing the cases out clarifies the intent.